docs/src/main/tut/xor.md

```diff
@@ -7,3 +7,338 @@ scaladoc: "#cats.data.Xor"
 ---
 # Xor
 
+In day to day programming, it is fairly common to find ourselves writing functions that
+can fail. For instance, querying a service may result in a connection issue, or some
+unexpected JSON response.
+
+To communicate these errors it has become common practice to throw exceptions. However,
+exceptions are not tracked in any way, shape, or form by the Scala compiler. To see
+if a function throws an exception, and moreover what kind of exception it throws, we
+have to dig through the source code. Then to handle these exceptions, we have to make sure
+we catch them at the call site. This all becomes even more unwieldy when we try to compose
+exception-throwing procedures.
+
+```scala
+val throwsSomeStuff: Int => Double = ???
+
+val throwsOtherThings: Double => String = ???
+
+val moreThrowing: String => List[Char] = ???
+
+val magic = throwsSomeStuff.andThen(throwsOtherThings).andThen(moreThrowing)
+```
+
+Assume we happily throw exceptions in our code. Looking at the types, any of those functions
+can throw any number of exceptions, we don't know. When we compose, exceptions from any of
+the constituent functions can be thrown. Moreover, they may throw the same kind of exception
+(e.g. `IllegalArgumentException`) and thus it gets tricky tracking exactly where that
+exception came from.
+
+How then do we communicate an error? By making it explicit in the data type we return.
+
+## Xor
+
+### Why not `Either`
+`Xor` is very similar to `scala.util.Either` - in fact, they are isomorphic.
```

Comment:
Maybe a quick gloss on what this means?

"... in fact they are *isomorphic* (that is, any `Either` value can be rewritten as an `Xor` value, and vice versa)."